english/staypuft/carrot-nose/find-carrot-nose.md

```diff
@@ -0,0 +1 @@
+Your snowman has been built, but he can't breathe! He needs a nose, STAT! You look around desperately, pushing aside the thick marshmallows, straining to see any flashes of orange. You can't find a carrot and you're running out of time!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.